modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1775,9 +1804,11 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
         throw new Error('specified key recovery service does not support recoveries for this coin');
       }
 
+      // check whether key material and password authenticate the users and return parent keys of all three keys of the wallet
       const keys = yield self.initiateRecovery(params);
 
-      const baseKeyPath = deriveKeys(deriveKeys(keys, 0), 0);
+      const defaultPrefix = 'm/0/0';
```

Comment:
Typically in BIP44 the `purpose` and `coinType` values are hardened (`account` is also, but that would be the *next* value in the path, which I don't see). I need to learn about initial bitgo wallet creation -- do we use un-hardened values like this as the root path?

Reply:
It seems that we do use unhardned values. 
What we used to have here, `deriveKeys(deriveKeys(keys, 0), 0)`, is the equivalance of deriving directly from `m/0/0`, based on the implementation in `bitgo-utxo-lib`.

Reply:
Just realized that the `derivePath` function only works for deriving from master node, not child nodes. Then we would still need to use the `derive()` function. I would have to first find out the different possibilities for key paths before proceeding here.